operations/services.md

```diff
@@ -312,6 +325,12 @@ Adds availability blocks which are used to group related [Availability update](#
 | --- | --- | --- | --- |
 | `AvailabilityBlocks` | array of [Availability block](#availability-block) | required | Availability blocks. |
 
+### Reservation purpose
```

Comment:
nitpick: Shouldn't this be somewhere closer to reservation? Like origin and other enums... in the end, this will be field on reservation as well, no?